.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
 .idea
 .build
+publis/js/_config.js
+public/js/components
```

Comment:
because postinstall will generate the components directory via the `copy-browser-modules` task and we don't want it in version control